src/services/alert-status.js

```diff
@@ -31,8 +31,10 @@ export function configureAlert(status, clearAlert) {
   } else if (status === -4) {
     alertObj.alertStyle = 'warning';
     alertObj.message = PEAKSJS_REACHED_END_OF_FILE;
-  } else {
+  } else if (status === 0) {
```

Comment:
I'd maybe stay away from using `0` or `1` as status codes.  They could get coerced to `true` or `false` values and cause side-effects.

Reply:
Yeah, good point. I will fix it.